src/components/ShortcutInput.tsx

```diff
@@ -38,23 +37,10 @@ export function ShortcutInput({ popoverAlign = 'center' }: ShortcutInputProps) {
   const [isOpen, setIsOpen] = useState(false)
   const [activeModifiers, setActiveModifiers] = useState<ModifierKey[]>([])
   const [activeKey, setActiveKey] = useState<string | null>(null)
-  const [currentShortcut, setCurrentShortcut] = useState('')
   const [recordingOpacity, setRecordingOpacity] = useState(1)
   const [snapshot, setSnapshot] = useState<KeySnapshot | null>(null)
   const loadShortcutFromStore = useShortcutStore((state) => state.loadShortcutFromStorage)
-
-  useEffect(() => {
```

Comment:
Do we not need to load up the shortcut anymore or is this being done in the shortcutStore?